examples/dev/slicing/ebi_empiar_3D_with_labels.py

```diff
@@ -0,0 +1,33 @@
+import napari
+import pooch
+from tifffile import imread
+
+"""
+This data comes from the MitoNet Benchmarks.
+
+Six benchmark volumes of instance segmentation of mitochondria from diverse volume EM datasets
+Narayan K , Conrad RW
+DOI: https://dx.doi.org/10.6019/EMPIAR-10982
+
+Data is stored at EMPIAR and can be explored here: https://www.ebi.ac.uk/empiar/EMPIAR-10982/
+
+With respect to the napari async slicing work, this dataset is small enough that it performs well in synchronous mode.
+"""
+
+salivary_gland_em_path = pooch.retrieve(
```

Comment:
How long should I expect these downloads to take? I've been waiting for over 5 minutes, just for this first one, and I'm not sure if something is stuck. Ideally I would get some measure of progress (e.g. with `tqdm`).

Reply:
Using `progressbar=True` now for a nice tqdm status bar with data sizes (1.6GBx2).